docs/operating-scylla/nodetool-commands/tasks/set-ttl.rst

```diff
@@ -0,0 +1,16 @@
+Nodetool tasks set-ttl
+======================
+**tasks set-ttl** - Sets task_ttl value in seconds, i.e. time for which tasks are kept in task manager after
+they are finished.
+
+Syntax
+-------
+.. code-block:: console
+
+   nodetool tasks set-ttl <time_in_seconds>
```

Comment:
Why is this a first class command rather than an option, like `nodetool tasks ttl --set <time_in_seconds>`?

Also need to document that this is set only in memory.
And refer to how to configure ttl in scylla configuration.

Reply:
I think details like that should be left to the online documentation. The inline documentation should be a quick reference only, with a link to the detailed documentation.
But I agree we need to mention these.

Reply:
ok, we can live with that documented only in the public doc and not in the quick reference (command help)

Reply:
The online documentation is also added in this PR so the comment is valid, it is just added to the wrong code-line. :)